### PR TITLE
add configuration to logout

### PR DESCRIPTION
**Pull request by domivds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,14 @@ cognizone:
...
             - view
 ```
-
+## Configure Logout
+```yaml
+cognizone:
+  security:
+    logout:
+      url: /api/logout    # URL to use to trigger logout, 
+      success-url: https://www.gruutemet.be/  # [OPTIONAL] URL to go to after successful logout (logout URL has to be set)
```
Comment (ssamarid) [inline README.md]: 🙄

**Review by ssamarid — APPROVED** [review]

**Review by alexey-lukashov — APPROVED** [review]
long-awaited feature, I remember it in the backlog about 2 years ago or mb even more

File: README.md (hunk 1 above)
Comment (alexey-lukashov) [reply README.md]: :joy:

**Review by christinaspachiou — APPROVED** [review]

File: security/src/main/java/zone/cogni/lib/security/common/PermissionGlobalMethodSecurityConfiguration.java [hunk 2]
```diff
@@ -32,4 +34,11 @@ protected AccessDecisionManager accessDecisionManager() {
     permissionService.ifPresent(service -> manager.getDecisionVoters().add(new PermissionAccessDecisionVoter(service)));
     return manager;
   }
+
+  @ConfigurationProperties(prefix = "cognizone.security")
```
Comment (natan-cox-cognizone) [inline security/src/main/java/zone/cogni/lib/security/common/PermissionGlobalMethodSecurityConfiguration.java]: @domivds naming convention is not bad: company / module / ... but then I am missing maybe a bit more context. I guess there can also be other things in `cognizone.security`. 

so why not make it `cognizone.security.global-properties`?

**Review by natan-cox-cognizone — APPROVED** [review]

File: security/src/main/java/zone/cogni/lib/security/common/PermissionGlobalMethodSecurityConfiguration.java (hunk 2 above)
Comment (domivds) [reply security/src/main/java/zone/cogni/lib/security/common/PermissionGlobalMethodSecurityConfiguration.java]: global = default = root?
but future-wise having it in a sub property will be cleaner :-)

**Pull request merged**